src/shrutiS/CustomerBankInfo.java

```diff
@@ -0,0 +1,163 @@
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+   a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+   b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+   c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                          Total debit operation - 5 times
+   d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		         SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+   e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+ */
+package shrutiS;
+
+import java.util.Scanner;
+
+public class CustomerBankInfo {
+
+	// Array with balance of banks in the following order ICICI, HDFC, BOB, SBI
+	double[] balance = new double[4];
+	// Array with credits of 4 banks in the following order ICICI, HDFC, BOB, SBI
+	int noOfCredits[] = new int[4];
+	// Array with debits of 4 banks in the following order ICICI, HDFC, BOB, SBI
+	int noOfDebits[] = new int[4];
```

Comment:
use of array here look we are increases complexity, if assume for there are 4 bank only but in future if it increases then to maintain this becomes difficult.

Reply:
when we want some data then we ahve to iterate this array put some condition, right?
if you make this as separate variable then then it very easy to access.